packages/react-dom/src/server/ReactPartialRenderer.js

```diff
@@ -187,7 +194,7 @@ function warnNoop(
     const constructor = publicInstance.constructor;
     const componentName =
       (constructor && getComponentName(constructor)) || 'ReactClass';
-    const warningKey = `${componentName}.${callerName}`;
```

Comment:
This change is because the previous version was screwing up my editor's syntax highlighting 😆

Reply:
Sounds like the worst possible reason for a change in a project with more than one developer.

Reply:
I can think of worse ones